### ISSUE TITLE
slidingSubmenus: false not working correctly together with linked parent

### ISSUE THREAD

**Issue by fritzmg** [FrDH/mmenu-js#1044]
Assume you have the following HTML markup:
```html
<nav id="mmenu">
  <ul class="level_1">
    <li class="active first"><span class="active first">Home</span></li>
    <li class="sibling">
      <a href="article.html" title="Article" class="sibling"><span>Article</span></a>
    </li>
    <li class="submenu sibling">
      <a href="page-1.html" title="Page 1" class="submenu sibling"><span>Page 1</span></a>
      <ul class="level_2">
        <li class="submenu first last">
          <a href="subpage-1.html" title="Subpage 1" class="submenu first last"><span>Subpage 1</span></a>
          <ul class="level_3">
            <li class="first last">
              <a href="subsubpage-1.html" title="Subsubpage 1" class="first last"><span>Subsubpage 1</span></a>
            </li>
          </ul>
        </li>
      </ul>
    </li>
    <li class="sibling last">
      <a href="archive.html" title="Archive" class="sibling last"><span>Archive</span></a>
    </li>
  </ul>
</nav>
```
i.e. a menu with submenu - but the menu item containing children also has a link.

Plus you use `slidingSubmenus: false` in your options.

mmenu does not seem to handle this case correctly, because when you open the submenu, it will look like this:

![mmenu](https://user-images.githubusercontent.com/4970961/62851324-a09dc800-bcdd-11e9-89c5-d698834c7b26.png)

For some reason, the arrow will now be its own entity and since it is not positioned absolutely, it will simply come after the submenu, when open.

**Comment by FrDH** [comment]
Thank you for bringing this to my attention. I'll see if I can find a fix shortly.

**Comment by FrDH** [comment]
Should be fixed in version 8.2.3

**Comment by fritzmg** [comment]
Thx for the quick fix :)
